frontend/src/strings/loc.ts

```diff
@@ -31,10 +31,11 @@ export function useTransliterate(): (key: string) => string {
 
     return (key: string): string => {
         let result = '';
-
+        let translitedChar;
         for (let i = 0; i < key.length; i++) {
             const char = key[i];
-            result += translit_ru_en[char] || char;
+            translitedChar = translit_ru_en[char];
```

Comment:
move `let` here

Reply:
Is it ok to declare a variable inside the loop?

Reply:
yes

Reply:
as you can see on line 26 there is a variable declared in the loop
`const char = key[i];`